ruby/ql/lib/change-notes/2024-02-15-activerecord_connection_sql_sinks

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
This file seems to have been mistakenly committed in addition to the change-note markdown file.